ingestr/src/sources.py

```diff
@@ -997,6 +997,77 @@ def dlt_source(self, uri: str, table: str, **kwargs):
         ).with_resources(endpoint)
 
 
+class TikTokSource:
+    # tittok://?access_token=<access_token>&advertiser_id=<advertiser_id>
+    def handles_incrementality(self) -> bool:
+        return True
+
+    def dlt_source(self, uri: str, table: str, **kwargs):
+        endpoint = "custom_reports"
+
+        parsed_uri = urlparse(uri)
+        source_fields = parse_qs(parsed_uri.query)
```

Comment:
Can tiktok access token contain `=`, `+` or `&` ? We may need to escape those.

Reply:
Regarding this I am not sure. We have not escaped them before, so we need to discuss this with the team.

Reply:
I agree with leaving this as is.